### PR TITLE
[Doc Day] Contribution Guide

### PR DESCRIPTION
**Pull request by spencerp**
**Patch description**
Adding some instructions for creating a pull request, in an attempt to make new contributors feel more comfortable submitting a PR.

I may have plagiarized a couple things from https://pytorch.org/docs/stable/community/contribution_guide.html

Also slipped in an unrelated typo fix.

**Testing steps**
```
make html
```
Screen Shot 2020-09-02 at 6 09 50 PM

**Logs**
n/a

**Other information**
none

**Data tests (if applicable)**
n/a

### CODE REVIEW

File: docs/source/cli_advanced.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Advance Scripts
+# Advanced Scripts
```
Comment (klshuster) [inline docs/source/cli_advanced.md:1]: thanks for catching this

File: docs/source/tutorial_tipsntricks.md [hunk 2]
```diff
@@ -123,3 +123,40 @@ git repository.
 
 For more detailed instructions and features, see the
 [README](http://github.com/facebookresearch/ParlAI/blob/master/example_parlai_internal)
+
+Contributing
+------------
+
+ParlAI is maintained by a small team at Facebook, so we rely heavily on community contributions. We welcome pull requests with open wings!
```
Comment (klshuster) [inline docs/source/tutorial_tipsntricks.md:130]: interestingly, I'm not sure if we ever mention that we're a team at Facebook explicitly, though I suppose the url `github.com/facebookresearch/ParlAI` gives it away

File: docs/source/tutorial_tipsntricks.md [hunk 3]
```diff
@@ -123,3 +123,40 @@ git repository.
...
+### Overview
+- Make your changes
+- Open a pull request
+- Iterate with reviewers, fix CI, add tests
+- Celebrate!
+
+#### Creating a Pull Request
+
+##### Selecting reviewers
+There’s no master gatekeeper for ParlAI, so don’t worry about needing to add any particular person. Just take a look at the blame for the file you’re modifying and try to find someone who has made a significant and recent change to that code. Also, add people about who might be affected by your change.
```
Comment (klshuster) [inline docs/source/tutorial_tipsntricks.md:141]: nit: "Also, add people who might be affected..." (extraneous "about")

**Review by klshuster — APPROVED** [review]

File: docs/source/tutorial_tipsntricks.md (hunk 2 above)
Comment (spencerp) [reply docs/source/tutorial_tipsntricks.md:130]: If we'd prefer to not put ourselves in a box, I could change this to "ParlAI is maintained by a small team, ..."

Comment (klshuster) [reply docs/source/tutorial_tipsntricks.md:130]: works for me! also more the truth, as we've had major contributors who never worked at Facebook

File: docs/source/tutorial_tipsntricks.md [hunk 4]
```diff
@@ -123,3 +123,40 @@ git repository.
...
+
+ParlAI is maintained by a small team at Facebook, so we rely heavily on community contributions. We welcome pull requests with open wings!
+
+### Overview
+- Make your changes
+- Open a pull request
+- Iterate with reviewers, fix CI, add tests
+- Celebrate!
+
+#### Creating a Pull Request
```
Comment (stephenroller) [inline docs/source/tutorial_tipsntricks.md:138]: Based on the left side TOC, I suggest making all of them ###

**Review by stephenroller — APPROVED** [review]

File: docs/source/tutorial_tipsntricks.md (hunk 4 above)
Comment (spencerp) [reply docs/source/tutorial_tipsntricks.md:138]: I fixed them to be consistent. I see that #### is a little over-indented. But if I make them ### they are almost the same size as the section header "Contributing" :/

Let me know if you feel strongly and I'll switch them to ###

**Pull request merged**